Index: Makefile
===================================================================
RCS file: /cvs/ports/editors/featherpad/Makefile,v
retrieving revision 1.14
diff -u -p -r1.14 Makefile
--- Makefile	17 Mar 2020 13:03:29 -0000	1.14
+++ Makefile	30 Apr 2020 17:16:28 -0000
@@ -2,12 +2,11 @@
 
 COMMENT =	lightweight Qt5 plain-text editor
 PKGNAME =	featherpad-${GH_TAGNAME:S/^V//}
-REVISION =	0
 CATEGORIES =	editors x11
 
 GH_ACCOUNT =	tsujan
 GH_PROJECT =	FeatherPad
-GH_TAGNAME =	V0.13.1
+GH_TAGNAME =	V0.14.0
 
 MAINTAINER =	Brian Callahan <bcallah@openbsd.org>
 
Index: distinfo
===================================================================
RCS file: /cvs/ports/editors/featherpad/distinfo,v
retrieving revision 1.12
diff -u -p -r1.12 distinfo
--- distinfo	17 Mar 2020 13:03:29 -0000	1.12
+++ distinfo	30 Apr 2020 17:16:28 -0000
@@ -1,2 +1,2 @@
-SHA256 (FeatherPad-0.13.1.tar.gz) = NOOdArumZDcYJIiOghc7rnmSXBCnLwYC/dfWS4xkt5A=
-SIZE (FeatherPad-0.13.1.tar.gz) = 822687
+SHA256 (FeatherPad-0.14.0.tar.gz) = MTNoVuWA9aPQxV3pk1/oGNGDYFa2oTgWinQFdRywVzs=
+SIZE (FeatherPad-0.14.0.tar.gz) = 827431
